ageevtask2/ageevtask2/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace ageevtask2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите радиус окружности : ");
+            double r = double.Parse(Console.ReadLine());
+            double l = Math.Round((2 * Math.PI * r), 3);
+            double S = Math.Round((Math.PI * Math.Pow(r, 2)), 3);
+            Console.WriteLine($"Длина окружности равна {l}, площадь равна {S}");
```

Comment:
В задании не требовалось округлять результаты. А вот при выводе  можно отформатировать, ограничив число знаков после запятой. 
Локальные переменные принято называть с маленьких букв.
Название переменной `l` не слишком удачное. Легко визуально спутать с `1`  или `I`.
И старайтесь давать переменным значимые имена, чтобы было понятно, какие величины из предметного мира задачи за ними стоят.
 Не стоит использовать `Math.Pow()` для возведения в квадрат. Используйте умножение.
```suggestion
            double raduis = double.Parse(Console.ReadLine());
            double length = 2 * Math.PI * raduis;
            double area = Math.PI * raduis * raduis;
            Console.WriteLine($"Длина окружности равна {length:F3}, площадь равна {area:F3}");
```